packages/frontapp/src/app/components/BookInfo/index.tsx

```diff
@@ -383,16 +584,29 @@ const BookInfo: FC<IBookInfoProps> = ({
             </WrapperButtons>
           )}
         </ShortDescriptionWrapper>
-        <LongDescription>
-          <Topic>Description: </Topic>
-          <Description>
-            {description ||
-              ' Lorem ipsum dolor sit amet consectetur, adipisicing elit. Sequi impedit aliquid alias consequuntur! Totam sequi expedita sunt dolor obcaecati, iusto ducimus? Beatae ea, commodi ab repellat, corporis atque quasi, tempore sunt modi similique soluta nemo hic necessitatibus esse accusantium omnis neque rerum. Placeat tempore, fugiat unde consequuntur dolor tempora ducimus.'}
-          </Description>
-          {userRole === RolesTypes.READER ? (
-            <OpenLink>see full description</OpenLink>
-          ) : null}
-        </LongDescription>
+        {editing ? (
+          <>
+            <br />
+            <TitleHolder>Description: </TitleHolder>
+            <WrapperInput>
+              <StyledTextArea
+                value={newDescription}
+                onChange={handleChangeDescription}
+              />
+            </WrapperInput>
+          </>
+        ) : (
+          <LongDescription>
+            <Topic>Description: </Topic>
+            <Description>
+              {description ||
+                ' Lorem ipsum dolor sit amet consectetur, adipisicing elit. Sequi impedit aliquid alias consequuntur! Totam sequi expedita sunt dolor obcaecati, iusto ducimus? Beatae ea, commodi ab repellat, corporis atque quasi, tempore sunt modi similique soluta nemo hic necessitatibus esse accusantium omnis neque rerum. Placeat tempore, fugiat unde consequuntur dolor tempora ducimus.'}
+            </Description>
+            {userRole === RolesTypes.READER ? (
+              <OpenLink>see full description</OpenLink>
+            ) : null}
```

Comment:
{ userRole === RolesTypes.READER && <>...</> } And what if manager also want to see full description?

Reply:
@x8er its done by design apparently, manager initially sees the whole description

Reply:
@x8er its done by design apparently, manager initially sees the whole description